iseq.c

```diff
@@ -1067,16 +1061,6 @@ rb_iseq_method_name(const rb_iseq_t *iseq)
     }
 }
 
-void
-rb_iseq_code_location(const rb_iseq_t *iseq, int *beg_pos_lineno, int *beg_pos_column, int *end_pos_lineno, int *end_pos_column)
```

Comment:
could you remain this function to access code location information?
This function is provided as version-specific not matured API.

Reply:
I see, patch updated.  Is there anything else you want to keep?

Reply:
No others. Thank you.